conan/tools/cmake/toolchain/blocks.py

```diff
@@ -563,6 +563,9 @@ class ExtraFlagsBlock(Block):
         {% if exelinkflags %}
         string(APPEND CONAN_EXE_LINKER_FLAGS "{% for exelinkflag in exelinkflags %} {{ exelinkflag }}{% endfor %}")
         {% endif %}
+        {% if defines %}
+        add_definitions({% for define in defines %} {{ define }}{% endfor %})
```

Comment:
Maybe we should use: https://cmake.org/cmake/help/latest/command/add_compile_definitions.html#command:add_compile_definitions

Reply:
Fine for me but the thing is that we're using `add_definitons` on more sides (blocks/toolchain)... Should we consider changing all of them in another PR? Let me know your opinion.

Reply:
Yes, good for me, leave the ``add_definitions()`` by now, and then later (create an issue) lets modernize all occurences of add_definitions.

Reply:
For the record 😉  -> https://github.com/conan-io/conan/issues/10929